nemo/collections/common/prompts/canary.py

```diff
@@ -15,13 +15,12 @@
 from typing import Any
 
 import torch
-from lhotse import CutSet, MonoCut
-from lhotse.cut import MixedCut
+from lhotse import MonoCut
+from lhotse.cut import Cut, MixedCut
 from lhotse.utils import ifnone
 
-from nemo.collections.common.prompts.fn import registered_prompt_format_fn
+from nemo.collections.common.data.prompt_fn import registered_prompt_format_fn
```

Comment:
Why move the file outside of the `prompts` folder?

Reply:
To avoid a circular dependency between prompt formatter APIs and data type APIs. This way we can define default prompt_format_fns right next to data type such as `SourceTargetTextExample` or `NeMoMultimodalConversation`.